RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,65 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation, and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, where we’d like to grow in future. 
+
+
+## Core Strategy Statement:  
```

Comment:
```suggestion
## Core Strategy Statement  
```

Reply:
From what I've seen in Google, we don't seem use colons in headings, although I can't find a good reference. This is the closest: https://developers.google.com/style/headings#heading-and-title-format
> Use punctuation in headings sparingly, if at all. Punctuation can be a sign that your heading is too complicated. Consider rewriting.

Reply:
Thought I got them all. Good catch. Thanks